pkg/controller/managedcluster/managedcluster_controller.go

```diff
@@ -632,5 +645,20 @@ func isAPIExtensionV1(managedClusterClient client.Client,
 	}
 	klog.V(4).Infof("isV1: %t", isV1 == -1)
 	return isV1 == -1, nil
+}
 
+func isAPIExtensionV1Only(managedCluster *clusterv1.ManagedCluster) (bool, error) {
+	var actualVersion string
+	//Search kubernetes version for creating manifestwork
+	if managedCluster.Status.Version.Kubernetes == "" {
+		return false, fmt.Errorf("kubernetes version not yet available for managed cluster %s", managedCluster.GetName())
+	}
+	actualVersion = managedCluster.Status.Version.Kubernetes
+	klog.V(4).Infof("actual kubernetes version is %s", actualVersion)
+	isV1, err := v1OnlyAPIExtensionMinVersion.Compare(actualVersion)
+	if err != nil {
+		return false, err
+	}
+	klog.V(4).Infof("isV1: %t", isV1 == -1)
+	return isV1 == -1, nil
```

Comment:
CRD v1beta1 is removed in kube 1.22. So `return isV1 == -1` or `return isV1 <= 0`?

Reply:
I think it should be `isV1 == -1`,  because
the result `-1` means the current kube version is less than 1.22.0, `0` means the current kube version equal to 1.22.0.
we should exclude the 1.22.0, am I right?

Reply:
fixed